### ISSUE TITLE
HTML generator produces faulty links for `instantiate` types

### ISSUE THREAD

**Issue by frank-aurich** [ndsev/zserio#242]
Using zserio 2.0.0, when using the `instantiate` keyword to instantiate template types, the HTML emitter creates faulty links to the instantiated elements.
Example:
```
package footest;

struct MyTemplate<T>(bool param)
{
   varuint16 x;
   T y if param;
}; 

instantiate MyTemplate<string> FooBar;
 
struct Foo
{
   FooBar(true) foobar;
}; 
```

When looking at type `Foo` in the generated HTML, clicking on the link to `FooBar` within the struct definition produces a "File Not Found" page, because within the struct `FooBar` is linked to `foo_test/content/footest/FooBar_TYPE_REFERENCE.html`, but the actual file that should be referred to is `foo_test/content/footest/FooBar_STRUCTURE.html`

**Comment by mikir** [comment]
Many thanks to point this out!

We are currently working on the HTML emitter (#241 Add markdown documentation comments into the language), so we will fix it soon.

**Comment by mikir** [comment]
This issue has been fixed within issue #156 (Extend documentation emitter to show templates).
